src/codegen/array_boundary.rs

```diff
@@ -0,0 +1,63 @@
+use super::vartable::Vartable;
+use crate::codegen::cfg::{ControlFlowGraph, Instr};
+use crate::codegen::Expression;
+use crate::sema::ast::Type;
+use indexmap::IndexMap;
+use num_bigint::{BigInt, Sign};
+use solang_parser::pt::Loc;
+
+// IndexMap <ArrayVariable res , (res of temp variable, expression of temp variable)>
+pub type ArrayTempVars = IndexMap<usize, (usize, Expression)>;
+
+//function to modify arraye length temp
```

Comment:
Add a space after the double slash. Don't forger to start your comment with a capital letter.

Reply:
Do so for all the other comments in your code.